app/kubernetes/views/applications/edit/components/ingress-table/ingress-table.controller.js

```diff
@@ -0,0 +1,27 @@
+export default class KubernetesApplicationIngressController {
+  /* @ngInject */
+  constructor($async, KubernetesIngressService) {
+    this.$async = $async;
+    this.KubernetesIngressService = KubernetesIngressService;
+  }
+
+  $onInit() {
+    return this.$async(async () => {
+      this.hasIngress;
```

Comment:
what does this line do?

Reply:
check the service has ingress associated with it or not